testing/e2e/suites/defaultdomain.go

```diff
@@ -240,6 +241,53 @@ func defaultDomainTests(in infra.Provisioned) []test {
 					return fmt.Errorf("parsing and verifying TLS certificate: %w", err)
 				}
 
+				lgr.Info("Validating Rotation Story")
+				certPEM, keyPEM, err := manifests.GenerateSelfSignedCert()
```

Comment:
name these variables appropriately to make it clear they're new

rotatedCert, rotatedKey := ...